head/src/main/resources/db/liquibase-changelog.xml

```diff
@@ -40,4 +40,6 @@
   <include file="changelog/creation/14-create-report.xml" relativeToChangelogFile="true"/>
   <include file="changelog/creation/15-create-data-component.xml" relativeToChangelogFile="true"/>
   <include file="changelog/creation/16-create-data-component-data-series.xml" relativeToChangelogFile="true"/>
+  <include file="changelog/creation/17-create-report-task.xml" relativeToChangelogFile="true"/>
+  <include file="changelog/creation/18-create-report-file.xml" relativeToChangelogFile="true"/>
```

Comment:
```suggestion
  <include file="changelog/creation/18-report-file.xml" relativeToChangelogFile="true"/>
```

Reply:
@ericjesse why should I take out the "create" here while other xml log files have it.

Reply:
I'm wrong, it is everywhere else, you are right. Forget my comment.